Dialogue/Afin/np_010030.csv

```diff
@@ -19,70 +19,70 @@ Afin_favs_present_080#0,"""Received \""Homing Emission\"" from Afin."""
 Afin_18_020#0,"""……なーんて、案外その辺りを<br>ふつーに歩いてたりしてな。<br>もう会っちゃってたり、とか？"""
-11_voice_NPC_5003_109#1,"""直接アークスシップが襲われたのに<br>みんな慌ててないって不思議だよな。<br>よくあること、とか言われる始末。"""
-Afin_19_020#0,"""……それを普通と思っちまうのって<br>なんかいろいろおかしくなってる証拠の<br>ような気がするんだよなあ。"""
-11_voice_NPC_5003_104#5,"""おれは別に<br>誰かを守りたいって気持ちで<br>アークスになったわけじゃないけど……"""
-Afin_20_020#0,"""やっぱ、ここを攻められたりすると<br>守りたいって自然と思うんだな。<br>なんか、自分自身がすっげえ不思議。"""
-11_voice_NPC_03_104#0,"""ダークファルスが復活したからか<br>アークス全体の空気が重いよなぁ。"""
-Afin_mb9_020#0,"""まあ、当たり前っちゃ当たり前だけど<br>へこんでばっかいても仕方がないと<br>おれは思うぜ。"""
-11_voice_NPC_03_106#0,"""ウワサの暴走龍ってさ<br>なんか結構数いるみたいだな。"""
-Afin_mb10_020#0,"""べらぼーに強いヤツもいるらしいから<br>気をつけた方がいいぜ。"""
-11_voice_NPC_03_104#1,"""まー、戦技大会にせよ何にせよ<br>騒いでうっぷんを晴らすってのは<br>いいことだと思うぜ。"""
-Afin_mb11_020#0,"""まわりでも結構参加する人が<br>多いみたいだし……<br>おれはどうしようかなぁ。"""
-11_voice_NPC_5003_105#3,"""…………<br>あっ、わ、悪い相棒。<br>ちょっと考え事してた。"""
-Afin_mb12_020#0,"""考えても何も解決しないって<br>そんなことわかってるんだけど……<br>おれ、切り替えが下手でさ。"""
-11_voice_NPC_5003_105#4,"""…………<br>あっ、わ、悪い相棒。<br>ちょっと考え事してた。"""
-Afin_mb13_020#0,"""考えても何も解決しないって<br>そんなことわかってるんだけど……<br>おれ、切り替えが下手でさ。"""
-11_voice_NPC_5003_104#6,"""惑星リリーパの採掘場跡、か。<br>……何かありそうな気がする。"""
+11_voice_NPC_5003_109#1,"""It's weird how the ARKS ship can be attacked directly and no one panics. I guess it happens too often for people to get upset."""
+Afin_19_020#0,"""...I feel like there is something wrong with you if you think that that is normal...。"""
+11_voice_NPC_5003_104#5,"""I didn't join ARKS because I wanted to protect someone."""
+Afin_20_020#0,"""Like I thought, if we get attacked here I end up thinking I want to protect everyone. It's weird to think about."""
+11_voice_NPC_03_104#0,"""Ever since the Dark Falz has been revived, everyone's been on edge."""
+Afin_mb9_020#0,"""Well, I might be stating the obvious but it's no use feeling down about it."""
+11_voice_NPC_03_106#0,"""Seems like there are a lot of rumours flying around."""
+Afin_mb10_020#0,"""There are some super strong people out there so be careful."""
+11_voice_NPC_03_104#1,"""Well, I think it's good to let off a little steam in the ARKS Battle Tournament every once and a while."""
+Afin_mb11_020#0,"""Looks like a lot of people are joining in...what should I do?"""
+11_voice_NPC_5003_105#3,"""...Ah, my bad partner. I was spacing out."""
+Afin_mb12_020#0,"""I know I can't solve anything just by thinking about it...but I'm bad at acting on my thoughts."""
+11_voice_NPC_5003_105#4,"""...Ah, my bad partner. I was spacing out."""
+Afin_mb13_020#0,"""I know I can't solve anything just by thinking about it...but I'm bad at acting on my thoughts."""
+11_voice_NPC_5003_104#6,"""The Quarry on Planet Lillipa....I feel like there's something there."""
 Afin_mb14_020#0,"""あ、別にお宝一攫千金、って<br>ことじゃないからな？<br>そんなの取り尽くされちゃってるだろ。"""
-11_voice_NPC_5003_104#7,"""……ほんと、冗談みたいだよな。<br>生き別れた姉がダークファルスって<br>どうなってんだよ……"""
+11_voice_NPC_5003_104#7,"""This...seems like a joke. My sister that I was seperated from is the Dark Falz...what's happening..."""
```

Comment:
There is a misspelling:
seperated or separated?

Reply:
I know English I swear... it's separated, will fix now.